app/controllers/devise/passwords_controller.rb

```diff
@@ -77,4 +82,27 @@ def unlockable?(resource)
     def translation_scope
       'devise.passwords'
     end
+
+    # Helper for use after calling send_*_instructions methods on a resource.
+    # If we are in paranoid mode, we always act as
+    # if the resource was valid and active for authentication
+    # and instructions were sent.
+    def successfully_sent?(resource)
```

Comment:
Reading the name of the method I expect just a `true` or `false` return.
But this method also set a flash message. Can we move flash manipulation from this method and to another?